backend/events/search-indexer/src/main/java/com/meemaw/search/indexer/KafkaUtils.java

```diff
@@ -6,7 +6,7 @@
 @UtilityClass
 public class KafkaUtils {
 
-  private final String BOOTSTRAP_SERVERS = "BOOTSTRAP_SERVERS";
+  private final String BOOTSTRAP_SERVERS = "KAFKA_BOOTSTRAP_SERVERS";
```

Comment:
What is the issue here? It was accepted before...

Reply:
@Meemaw

Reply:
Those tools do not play well with Lombok magic (`@UtilityClass`). I would remove `@UtilityClass` as I don't see it bringing enough value and then:
- make fields `final static`
- add a private constructor to the class